edgesite/README.md

```diff
@@ -0,0 +1,226 @@
+
+# EdgeSite: Standalone Cluster at edge
+
+## Abstract
+
+In Edge computing, there are scenarios where customers would like to have a whole cluster installed at edge location. As a result,
+admins/users can leverage the local control plane to implement management functionalities and take advantages of all edge computing's benefits.
+
+EdgeSite helps running lightweight clusters at edge.
+
+## Motivation
+
+There are scenarios user need to run a standalone Kubernetes cluster at edge to get full control and improve the offline scheduling capability. There are two scenarios user need to do that:
+
+* The edge cluster is in CDN instead of the user's site
+
+  The CDN sites usually be large around the world and the network connectivity and quality cannot be guaranteed. Another factor is that the application deployed in CDN edge do not need to interact with center usually. For those deploy edge cluster in CDN resources, they need to make sure the cluster is workable without the connection with central cloud not only for the deployed applicatons but also the schedule capabilities. So that the CDN edge is manageable regardless the connection to one center.
```

Comment:
The CDN sites usually be large  --> The CDN sites are usually large

Reply:
For those deploy edge cluster   --> For those who deploy edge cluster

Reply:
but also the schedule capabilities.  --> this doesn't sound correct. can you please make it more clear.